metadata_mapper/mappers/mapper.py

```diff
@@ -1110,11 +1111,16 @@ def capitalize_value(self, exclude):
             "sourceResource/contributor",
             "sourceResource/extent",
             "sourceResource/format",
+            "sourceResource/type",
```

Comment:
why was this added here?

Reply:
I'm sorry, I need to take better notes on changes like this next time. The explanation for this completely escapes me. I'll remove it and run this through the validator.

Reply:
It appears that the `type` field is lowercased, seemingly by another enrichment because the mapper returns the value "Image" with the first letter capitalized. My change was a way to make it match the value in Solr.

Reply:
It's the `enrich_type` enrichment that's doing it. Perhaps this is intentional, solr is wrong and the type should be lowercase.